README.md

```diff
@@ -130,11 +130,22 @@ Example: `partners_count: 4`
 Fractions of the original dataset each partner receives to mock a collaborative ML scenario where each partner provides data for the ML training.  
 Example: `amounts_per_partner: [0.3, 0.3, 0.1, 0.3]`
 
-`samples_split_option`: `random` or `stratified`  
+`samples_split_option`: `random` (default) or `stratified`  
```

Comment:
After some thought, why don't we keep the sample_split_option ? 
`samples_split_option`: `random` (default) or `stratified` or `[[nb of clusters (int), 'shared' or 'specific']]
I think it's more consistent to have only one parameter.

Reply:
Done